src/db.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require('mongoose');
+
+class DbConnection {
+  constructor(url) {
+    this.url = url;
+    this.connection = null;
+  }
+
+  async connect() {
+    try {
+      await mongoose.connect(this.url);
+      console.log(`Successfuly connected to ${this.url}`);
+    } catch (err) {
+      console.log(
+        `Connection to ${this.url} has failed. With error code: \n${err}`
+      );
+    }
+  }
+
+  getConnection() {
+    if (this.connection == null) {
+      this.connect();
+      this.connection = mongoose.connection;
+    }
+    return this.connection;
+  }
+}
+
+module.exports = DbConnection;
```

Comment:
You can also have a `closeConnection()` function and/or `dropDatabase()` function here which can be useful when you're writing tests to periodically close or clear the DB.